src/Fmt_ast.ml

```diff
@@ -1420,9 +1428,9 @@ and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext ({ast= exp} as xexp)
           ( wrap_if parens "(" ")"
               (hovbox 0
                  ( hovbox 2
-                     ( hovbox 2
+                     ( hvbox_if_sparse c.conf.fun_call 2
```

Comment:
Try changing 2 to 4 here or the line above I think.

Reply:
The only thing I came up with is https://github.com/ocaml-ppx/ocamlformat/pull/283/commits/a55ffe1823154a698820d6a3bd9207872541b0f2